products/greenlight.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Greenlight
+category: server-app
+iconSlug: bigbluebutton
+permalink: /greenlight
+releasePolicyLink: https://github.com/bigbluebutton/greenlight/security
```

Comment:
```suggestion
```

Let's document this directly in the description with a more informative sentence.

Reply:
I'm not sure about this one.
If the default sentence is not great, wouldn't it make more sense to improve it instead of rewriting it in every product page?

Reply:
It's not great only in a few cases, such as when there is only a table listing all versions with their status (such as https://github.com/bigbluebutton/greenlight/security). The vast majority on products on endoflife.date have more than that, and that default sentence is adapted to that case.

In this particular case I find it more informative to have this:

```
Greenlight supported versions are documented on <https://github.com/bigbluebutton/greenlight/security>.
```

Instead of this:

```
More information is available on the [Greenlight website](https://github.com/bigbluebutton/greenlight/security).
```

Reply:
Fair enough, you have a point.